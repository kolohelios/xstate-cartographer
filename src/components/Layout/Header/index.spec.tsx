import * as React from 'react'
import TestRenderer from 'react-test-renderer'
import { Header } from '.'

jest.mock('src/sampleMachines/defaultMachine.js.txt', () => {})

describe('Header', () => {
	it.todo('renders correctly', /*() => {
		const header = TestRenderer.create(<Header />).toJSON()
		expect(header).toMatchSnapshot()
	}*/)

})
