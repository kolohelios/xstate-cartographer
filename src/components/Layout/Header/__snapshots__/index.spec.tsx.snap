// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="sc-bdVaJa hFffZV"
>
  <p
    className="sc-bwzfXH iJxcRz"
  >
    XState Cartographer
  </p>
</div>
`;
